tests/Operation/FindAndModifyFunctionalTest.php

```diff
@@ -92,6 +93,25 @@ public function testHintOptionAndUnacknowledgedWriteConcernUnsupportedClientSide
         $operation->execute($this->getPrimaryServer());
     }
 
+    public function testFindAndModifyReportedWriteConcernError(): void
+    {
+        if (($this->isShardedCluster() && ! $this->isShardedClusterUsingReplicasets()) || ! $this->isReplicaSet()) {
+            $this->markTestSkipped('Test only applies to replica sets');
+        }
+
+        $this->expectException(CommandException::class);
+        $this->expectExceptionCode(100);
```

Comment:
Note: this is "UnsatisfiableWriteConcern" per https://github.com/mongodb/mongo/blob/master/src/mongo/base/error_codes.yml

Reply:
https://github.com/mongodb/mongo-php-library/pull/984